src/app/shared/components/theme-mode-toggle/theme-mode-toggle.component.html

```diff
@@ -0,0 +1,4 @@
+<button class="theme-mode-toggle" (click)="toggleThemeMode()">
+  <span class="material-icons" *ngIf="!isDarkMode"> light_mode </span>
+  <span class="material-icons" *ngIf="isDarkMode"> dark_mode </span>
+</button>
```

Comment:
Nitpick: Should the button show the current state, or the state that will be applied when it is toggled?

Reply:
> Nitpick: il pulsante dovrebbe mostrare lo stato corrente o lo stato che verrà applicato quando viene attivato o disattivato?

The button shows the current state of how the theme is configured, so clicking the button changes the state to either "dark" or "light" mode, while also changing the icon. By default it is set to light mode at Kamil's request (see previous comments).🙂

Reply:
Sure, what I mean essentially is that when a user opens the docs, and wants to switch to dark mode, they might search the page for the button to turn on dark mode. The button will show a sun, but the page is already in light mode.

I'm sure users will get it, I don't think there is any standard on whether to show the current state, or the opposite state on a button.

I think the current approach works, and I think we are ready to merge this PR.

Reply:
I hope too that it will be merged soon 🙏🏻, it is a nice feature for documentation then especially in the evening it is very useful if you want to read the documentation of NestJS 😻 😎